app/controllers/owners_controller.rb

```diff
@@ -0,0 +1,38 @@
+class OwnersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @owners = Owner.order(created_at: :desc)
+  end
+
+  def new
+    @user  = User.new
+    @owner = @user.build_owner
+  end
+
+  def create
+      @user  = User.new
+      @owner = @user.build_owner(owner_params)
+      # @owner.user_id = current_user.id
+
+      if @owner.save
+        flash[:notice] = "Owner created"
+        redirect_to @owner
+      else
+        render 'new'
+      end
+  end
+
+  def show
+    # @owner = current_user.owner
+    @owner = Owner.find(params[:id])
+  end
+
+
+  private
+    def owner_params
+      params.require(:owner).permit(:user_id, :first_name, :middle_name, :last_name,
```

Comment:
Since you're using the `build_owner` methods above, I don't think that you need to pass in the `user_id` here. It should come in automatically when you call `@user.build_owner`.

Reply:
Ah!! that makes sense. I'll need to get rid of `user_id` from the strong params. Thank you so much.